File: Unidad_2/01_Lambda_sin_parametros/02_Lambda_ejemplo/LambdaTest.java
public class LambdaTest{
    public static void main(String[] args) {
        //Expreción lambda => representa un objeto de una interfaz funcional
        FunctionTest ft = () -> System.out.println("Hola Mundo"); //implementacion del metodo "saludar()"
                                                                // de la Interfaz Funcional<
        //ft.saludar();

        LambdaTest objeto = new LambdaTest();
        objeto.miMetodo(ft);
    }

    public void miMetodo(FunctionTest parametro){
        parametro.saludar();
    }
}

File: 2 _oportunidad/_04_ejercicios/lambda_sin_parametros_v2/LambdaTest.py
class LambdaTest:
    @staticmethod
    def main(args):
        # Esprexion Lambda => represnetando un objeto de una Interfaz funcional (FunctionTest)
        ft = lambda : print("Hola Mundo")
        objeto = LambdaTest()
        objeto.miMetodo(ft)

    def miMetodo(self, parametro):
        parametro.saludar()
